plugins/bazaar/src/components/BazaarOverviewCard/BazaarOverviewCard.tsx

```diff
@@ -24,13 +24,16 @@ import type { BazaarProject } from '../../types';
 import { bazaarApiRef } from '../../api';
 import { fetchCatalogItems } from '../../util/fetchMethods';
 import { parseBazaarProject } from '../../util/parseMethods';
-import { ErrorPanel, InfoCard } from '@backstage/core-components';
+import { ErrorPanel, InfoCard, Link } from '@backstage/core-components';
 import { bazaarPlugin } from '../../plugin';
+import { IconButton } from '@material-ui/core';
+import StorefrontIcon from '@material-ui/icons/Storefront';
 
 /** @public */
 export type BazaarOverviewCardProps = {
   order: 'latest' | 'random';
-  fullWidth?: boolean;
+  fullCard?: boolean;
```

Comment:
makes me wonder if it hadn't been more clean to make this

```
width?: 'small' | 'large';
height?: 'small' | 'large';
```

:)

Arguably, the name `fullCard` is a step backwards in understandability of the name isn't it? If you're going to break things, maybe it's better to make it consistent and talk about width and height explicitly.

Reply:
Yes I've had some difficulties making up my mind but you are probably right. I'll go with fullHeight and fullWidth to keep it consistent 👍